.azure-pipelines.yml

```
@@ -0,0 +1,78 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- azure-pipelines
+- 2.13.x
+
+jobs:
+- job: CI_Build
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+  pool:
+    vmImage: 'Ubuntu-16.04'
+  steps:
+  - bash: bash -x init.sh
+    workingDirectory: $(Build.SourcesDirectory)/admin  
+    displayName: 'Run init.sh'
```

```suggestion
    displayName: 'Running init.sh'
```

You can still add comments outside of the suggestion block.